compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1449,7 +1449,7 @@ error.invalid.use.of.experimental.feature=\
   using experimental feature ''{0}''. use ''--experimental'' flag to enable the experimental features
 
 error.invalid.use.of.null.literal=\
-''null'' literal is only supported for ''json''
+''null'' literal is only allowed in a json-related context
```

Comment:
`json-related` -> `JSON-related`. The lower case is only used when referring to the `json` type.